packages/client/hmi-client/src/components/model/petrinet/tera-model-description.vue

```diff
@@ -45,72 +25,57 @@
+import type { Dataset, Model } from '@/types/Types';
 import TeraModelDiagram from '@/components/model/petrinet/model-diagrams/tera-model-diagram.vue';
 import TeraModelEquation from '@/components/model/petrinet/tera-model-equation.vue';
 import { isDataset, isModel, type Asset } from '@/utils/asset';
 import TeraProgressSpinner from '@/components/widgets/tera-progress-spinner.vue';
+import Editor from 'primevue/editor';
 
 const props = defineProps<{
 	model: Model;
 	featureConfig?: FeatureConfig;
 	isGeneratingCard?: boolean;
 }>();
 
-const emit = defineEmits(['update-model', 'model-updated']);
+const emit = defineEmits(['update-model']);
 const teraModelDiagramRef = ref();
 
-const card = computed<any>(() => props.model.metadata?.gollmCard ?? null);
-const summary = computed(() => card.value?.summary?.modelSummary ?? '');
-const description = computed(() => card.value?.details?.modelDescription ?? props.model?.header?.description ?? '');
-
-const biasAndRiskLimitations = computed(() => card.value?.biasRisksLimitations?.modelBiasRisksLimitations ?? '');
-const modelType = computed(() => card.value?.details?.modelType ?? props.model.header.schema_name ?? '');
-const fundedBy = computed(() => card.value?.details?.fundedBy ?? '');
-const evaluation = computed(() => card.value?.testing?.testingDataFactorsMetrics ?? '');
-const technicalSpecifications = computed(() => card.value?.specs?.modelSpecs ?? '');
-const glossary = computed(() => card.value?.glossary ?? []);
-const moreInformation = computed(() => card.value?.moreInformation?.links ?? []);
-
-const uses = computed(() => card.value?.uses ?? null);
-const authors = computed(() => {
-	const authorsSet: Set<string> = new Set();
-	if (props.model?.metadata?.annotations?.authors)
-		props.model.metadata.annotations.authors.forEach((author: Author) => authorsSet.add(author.name));
-	if (card.value?.authors) card.value.authors.forEach((author: string) => authorsSet.add(author));
-	return [...authorsSet].join(', ');
-});
-
+const currentActiveIndexes = ref([0, 1, 2, 3]);
 const relatedTerariumArtifacts = ref<Asset[]>([]);
 const relatedTerariumModels = computed(() => relatedTerariumArtifacts.value.filter((d) => isModel(d)) as Model[]);
 const relatedTerariumDatasets = computed(() => relatedTerariumArtifacts.value.filter((d) => isDataset(d)) as Dataset[]);
-</script>
 
-<style scoped>
-.description {
-	display: flex;
-	gap: var(--gap-small);
-	flex-direction: column;
-	grid-template-columns: max-content 1fr;
-	margin-left: var(--gap-6);
+// Editor for the description
+const editorContent = ref(props.model?.metadata?.description ?? '');
 
-	& label {
-		font-weight: bold;
-		margin-right: var(--gap-small);
+watch(editorContent, () => {
+	if (editorContent.value !== props.model?.metadata?.description) {
+		const updatedModel = {
+			...props.model,
+			metadata: { ...props.model.metadata, description: editorContent.value }
+		} as Model;
+		emit('update-model', updatedModel);
+	}
+});
```

Comment:
We may want to debounce this, even though the emit event changes/updates the "temporary model", the temporary-model is used to trigger other API calls so we are firing like 2-3 requests every key stroke.